Source/Runtime/Events.Processing/Filters.proto

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+syntax = "proto3";
+
+import "Fundamentals/Protobuf/Uuid.proto";
+import "Fundamentals/Protobuf/Failure.proto";
+import "Fundamentals/Services/ReverseCallContext.proto";
+import "Runtime/Events/Committed.proto";
+import "Runtime/Events.Processing/Processors.proto";
+
+package dolittle.runtime.events.processing;
+
+option csharp_namespace = "Dolittle.Runtime.Events.Processing.Contracts";
+
+message SuccessfulFilter {
+    bool isIncluded = 1;
+    protobuf.Uuid partitionId = 2;
+}
+
+message FiltersRegistrationRequest {
+    services.ReverseCallArgumentsContext callContext = 1;
+    protobuf.Uuid scopeId = 2;
+    protobuf.Uuid filterId = 3;
+}
+
+message FilterResponse {
+    services.ReverseCallResponseContext callContext = 1;
+    oneof Result {
+        SuccessfulFilter success = 2;
+        ProcessorFailure failed = 3;
+    }
+}
```

Comment:
This is a little inconsistent with how we use failures other places. I suggest you remove the oneof, and put the expected results from `SuccessfulFilter` directly in the response, alongside the `ProcessorFailure`, and call it `failure`. So that we do the same here - check if `failure` is set, and if not we assume success.

Something like:
 ```suggestion
message FilterResponse {
    services.ReverseCallResponseContext callContext = 1;
    bool isIncluded = 2;
    protobuf.Uuid partitionId = 3;
    ProcessorFailure failure = 4;
}
```

Reply:
Yes I agree. I'll do that